src/library-authoring/create-library/LibraryCreatePage.jsx

```diff
@@ -81,6 +83,17 @@ export class LibraryCreatePage extends React.Component {
 
     if (prevProps.status === SUBMISSION_STATUS.SUBMITTING && this.props.status === SUBMISSION_STATUS.FAILED) {
       window.scrollTo({ top: 0, left: 0, behavior: 'smooth' });
+
+      // Update field errors after submission with error response
+      if (this.props.errorFields) {
+        // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
Is there sense to rewrite `componentDidUpdate` to `getDerivedStateFromProps`? As I understand it can result in an infinite loop according to eslint error, doesn't it?

Reply:
Yes, you are right but we can't check previous props values with the getDerivedStateFromProps method. And for using it we need to create an additional methods and clone some props to state.